synthtool/gcp/gapic_bazel.py

```diff
@@ -88,6 +88,9 @@ def _generate_code(
             api_definitions_repo = self._clone_googleapis()
             api_definitions_repo_name = "googleapis"
 
+        # conditionally apply custom toolchain
+        _check_for_custom_kokoro_bazel_rules(api_definitions_repo)
```

Comment:
The naming is somewhat inaccurate. There are no custom kokoro rules, but there are either "custom build flags" or "custom toolchains".